static/app/views/dashboards/widgets/timeSeriesWidget/plottables/bars.tsx

```diff
@@ -1,36 +1,58 @@
 import Color from 'color';
+import type {BarSeriesOption, LineSeriesOption} from 'echarts';
 
 import BarSeries from 'sentry/components/charts/series/barSeries';
 
-import type {TimeSeries} from '../../common/types';
+import type {PlottableData} from '../../common/types';
 import {markDelayedData} from '../markDelayedData';
 import {timeSeriesItemToEChartsDataPoint} from '../timeSeriesItemToEChartsDataPoint';
 
-import {Plottable} from './plottable';
+import {
+  type AggregateTimePlottingOptions,
+  AggregateTimeSeries,
+} from './aggregateTimeSeries';
 
-interface BarsOptions {
+interface BarsConfig {
+  /**
+   * Optional color. If not provided, a backfill from a common palette will be provided to `toSeries`
+   */
```

Comment:
Is it worth extracting the `color` and `delay` interface into something we can union for our configs? That way we don't need to keep duplicating the explanations for these

Reply:
I definitely thought the same! I think it's _mostly_ fine as-is, but I agree that copying the descriptions is annoying. I think if there's _one more_ (though I can't imagine what it would be), it'll be worth extracting